### PR TITLE
Update broken OGP image

### PR DESCRIPTION
**Pull request by this-pama**

### CODE REVIEW

File: routes/helpers/parsers/index.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+const { app_storage, app_title_short } = require('../../../config/')
```
Comment (JosuaKrause) [inline routes/helpers/parsers/index.js:1]: use include and absolute path

**Comment by myjyby** [comment]
Hi @this-pama. Thanks for the work on this. Just FYI I am tweaking some things here to not have to rely on a new getPadImg function in routes/helpers/parsers and to fix the check for public pads in views/partials/head.ejs (the current code uses locals.metadata.page.public which only checks for public sessions, i.e. sessions without login, not for publicly visible pads).

**Comment by this-pama** [comment]
> Hi @this-pama. Thanks for the work on this. Just FYI I am tweaking some things here to not have to rely on a new getPadImg function in routes/helpers/parsers and to fix the check for public pads in views/partials/head.ejs (the current code uses locals.metadata.page.public which only checks for public sessions, i.e. sessions without login, not for publicly visible pads).

Yes, I can confirm after checking the `sessiondata` declaration that the `public` data point relates to the user session. What data point indicates if a pad is public? `publishable` ?

**Comment by myjyby** [comment]
OK finished my fixes. I created an "excerpt" object that gets sent from the backend (always declared in datastructures > metadata and updated for specific public content in routes/contribute/pad/ and routes/browse/pads/render.js). This is then used and slightly adapted contextually in views/partials/head.ejs.

File: routes/browse/pads/render.js [hunk 2]
```diff
@@ -246,7 +246,12 @@ module.exports = async (req, res) => {
 				tags: statistics.tags
 			}
 
-			const metadata = await datastructures.pagemetadata({ req, res, page, pagecount: Math.ceil((array.sum.call(statistics.filtered, 'count') || 0) / page_content_limit), map, display: pinboard_out?.slideshow && (!pinboard_out?.editable || activity === 'preview') ? 'slideshow' : display, mscale })
+
+			console.log(pinboard_out)
```
Comment (JosuaKrause) [inline routes/browse/pads/render.js:250]:
```suggestion
```

**Review by JosuaKrause — APPROVED** [review]
lgtm

**Pull request merged**